backend/webapi/Features/Banners/BannersController.cs

```diff
@@ -0,0 +1,19 @@
+namespace Pidp.Features.Banners;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+using Pidp.Infrastructure.Services;
+
+[Route("api/[controller]")]
+public class BannersController(IPidpAuthorizationService authorizationService) : PidpControllerBase(authorizationService)
+{
+    [HttpGet]
+    [AllowAnonymous]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
These `ProducesResponseType` attributes are for our swagger documentation, so don't affect functionality if they're incorrect. 
But if you look at the code, there is no way for this endpoint to generate a `404NotFound`, so we should remove this line.

Reply:
fixed.